server/datastore/mysql/hosts.go

```diff
@@ -4217,109 +4216,3 @@ func (ds *Datastore) GetMatchingHostSerials(ctx context.Context, serials []strin
 
 	return result, nil
 }
-
-func (ds *Datastore) NewHostScriptExecutionRequest(ctx context.Context, request *fleet.HostScriptRequestPayload) (*fleet.HostScriptResult, error) {
```

Comment:
Same as in `service` package, this refactor moves script-related code in a `scripts.go` file.